app/javascript/packs/component-definitions-common.js

```diff
@@ -37,6 +37,12 @@ import SetOwnershipForm from '../components/set-ownership-form';
 import TableListViewWrapper from '../react/table_list_view_wrapper';
 import TaggingWrapperConnected from '../components/taggingWrapper';
 import TextualSummaryWrapper from '../react/textual_summary_wrapper';
+import RightSection from '../components/top-navbar/right-section';
```

Comment:
Should we prefix these components names somehow?

I don't think `UserOptions`, `Help`, `Configuration` etc. are really meaningful names outside the navbar context.

What about registering all these as `navbar.UserOptions`, `navbar.Help`, etc.?